ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less

```diff
@@ -76,6 +76,24 @@
 		background: #efaa9b;
 	}
 }
+.btn-phone-call  {
+.cmp-button {
+        .btn();
+		background: #2979be;
+		border: 1px solid #165fab;
+		color: #fff;
+		&:hover {
+			background: #0c4f95;
+		}
+
+		&:before {
+    content: "\f095";
+    font: normal normal normal 14px/1 FontAwesome;
+    padding-right: 3px;
+}
+
+    }
+}
 
 .btn-ternary__disabled  {
     .cmp-button {
```

Comment:
@ReenaOmana : How is tel: tag working without making changes in CQ link Checker Osgi Config ?